### PR TITLE
Update URLs after renaming repository

### PR DESCRIPTION
**Pull request by torbsorb**

### CODE REVIEW

**Comment by eskaur** [comment]
Why is this needed?

**Comment by torbsorb** [comment]
> 
> 
> Why is this needed?
> Edit: I know it's CI related, but the CI already works on master. So just trying to understand.

We want the badge in the README.md to be relevant :)

**Comment by SatjaSivcev** [comment]
> 
> 
> > Why is this needed?
> > Edit: I know it's CI related, but the CI already works on master. So just trying to understand.
> 
> We want the badge in the README.md to be relevant :)

What is this badge? Sorry I accidentaly closed the PR.

**Comment by eskaur** [comment]
I assume it's what makes this badge work:
image

File: source/Applications/Advanced/HandEyeCalibration/README.md [hunk 1]
```diff
@@ -34,5 +34,5 @@ The following applications assume that a **Transformation Matrix** has been foun
   * AxisAngle, Rotation Vector, Roll-Pitch-Yaw, Quaternion
 
 [SampleHandEyeCalibration-url]: https://www.zivid.com/hubfs/softwarefiles/releases/1.6.0+7a245bbe-26/doc/cpp/zivid_sample_code.html#autotoc_md10
-[UtilizeEyeInHandCalibration-url]: https://github.com/zivid/cpp-extra-samples/blob/master/Applications/Advanced/HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp
-[PoseConversions-url]: https://github.com/zivid/cpp-extra-samples/blob/master/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp
\ No newline at end of file
+[UtilizeEyeInHandCalibration-url]: https://github.com/zivid/zivid-cpp-samples/blob/master/Applications/Advanced/HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp
```
Comment (knatten) [inline source/Applications/Advanced/HandEyeCalibration/README.md:37]: Do these need to absolute urls?  Seems we use relative ones in the other `README.md`s in this repo.

Comment (torbsorb) [reply source/Applications/Advanced/HandEyeCalibration/README.md:37]: I'll leave that for another PR. @chrisasc?

Comment (knatten) [reply source/Applications/Advanced/HandEyeCalibration/README.md:37]: https://github.com/zivid/zivid-cpp-samples/issues/116

**Pull request merged**